semver/parse_range_test.ts

```diff
@@ -0,0 +1,869 @@
+      ],
+    ]],
+    ["~>3.2.1", [
+      [
+        { operator: ">=", major: 3, minor: 2, patch: 1, prerelease: [] },
+        { operator: "<", major: 3, minor: 3, patch: 0 },
+      ],
+    ]],
+    ["~1", [
+      [
+        { operator: ">=", major: 1, minor: 0, patch: 0 },
+        { operator: "<", major: 2, minor: 0, patch: 0 },
+      ],
+    ]],
+    ["~>1", [
+      [
+        { operator: ">=", major: 1, minor: 0, patch: 0 },
+        { operator: "<", major: 2, minor: 0, patch: 0 },
+      ],
+    ]],
+    ["~1.0", [
+      [
+        { operator: ">=", major: 1, minor: 0, patch: 0 },
+        { operator: "<", major: 1, minor: 1, patch: 0 },
+      ],
+    ]],
+    ["<1", [[{ operator: "<", major: 1, minor: 0, patch: 0 }]]],
+    [">=1.2", [[{ operator: ">=", major: 1, minor: 2, patch: 0 }]]],
+    ["~v0.5.4-beta", [
+      [
+        {
+          operator: ">=",
+          major: 0,
+          minor: 5,
+          patch: 4,
+          prerelease: ["beta"],
+        },
+        { operator: "<", major: 0, minor: 6, patch: 0 },
+      ],
+    ]],
+    ["=0.7.x", [
+      [
+        { operator: ">=", major: 0, minor: 7, patch: 0 },
+        { operator: "<", major: 0, minor: 8, patch: 0 },
+      ],
+    ]],
+    [">=0.7.x", [[{ operator: ">=", major: 0, minor: 7, patch: 0 }]]],
+    ["<0.7.x", [[{ operator: "<", major: 0, minor: 7, patch: 0 }]]],
+    [">1.2", [[{ operator: ">=", major: 1, minor: 3, patch: 0 }]]],
+    ["^0.0.1", [
+      [
+        { operator: ">=", major: 0, minor: 0, patch: 1, prerelease: [] },
+        { operator: "<", major: 0, minor: 0, patch: 2 },
+      ],
+    ]],
+    ["^0.0.1", [
+      [
+        { operator: ">=", major: 0, minor: 0, patch: 1, prerelease: [] },
+        { operator: "<", major: 0, minor: 0, patch: 2 },
+      ],
+    ]],
+    ["^1.2.3", [
+      [
+        { operator: ">=", major: 1, minor: 2, patch: 3, prerelease: [] },
+        { operator: "<", major: 2, minor: 0, patch: 0 },
+      ],
+    ]],
+    ["^1.2.3", [
+      [
+        { operator: ">=", major: 1, minor: 2, patch: 3, prerelease: [] },
+        { operator: "<", major: 2, minor: 0, patch: 0 },
+      ],
+    ]],
+    ["^1.2", [
+      [
+        { operator: ">=", major: 1, minor: 2, patch: 0 },
+        { operator: "<", major: 2, minor: 0, patch: 0 },
+      ],
+    ]],
+    ["blerg", [
+      [
+        {
+          operator: "<",
+          major: 0,
+          minor: 0,
+          patch: 0,
+          prerelease: [],
+          build: [],
+          semver: { major: 0, minor: 0, patch: 0, prerelease: [], build: [] },
+        },
+      ],
+    ]],
+    ["^1.2.3", [
+      [
+        { operator: ">=", major: 1, minor: 2, patch: 3, prerelease: [] },
+        { operator: "<", major: 2, minor: 0, patch: 0 },
+      ],
+    ]],
+  ];
+
+  for (const [r, expected] of ranges) {
+    const range = parseRange(r);
+    assertEquals(range, expected, 'call to parseRange("' + r + '") failed');
+  }
+});
+
+Deno.test("parseHyphenRanges", () => {
```

Comment:
We have some rule for naming test cases: https://github.com/denoland/deno_std/blob/main/.github/CONTRIBUTING.md#tests

This case probably should be something like:
```suggestion
Deno.test("parseRange() parses ranges with hyphens", () => {
```

Can you also update the other test cases alike?

Reply:
done :)